app/models/version.rb

```diff
@@ -24,6 +24,18 @@ def self.reverse_dependencies(name)
       .where(rubygems: { name: name })
   end
 
+  def self.reverse_runtime_dependencies(name)
```

Comment:
Resisted the urge to be clever here and have a single `self.reverse_dependencies` method with an `only` argument set to `nil` by default and a conditional. Let me know if you'd prefer something like that over the slight duplication here.